homeassistant/components/comelit/climate.py

```diff
@@ -115,10 +115,10 @@ def __init__(
         # because no serial number or mac is available
         self._attr_unique_id = f"{config_entry_entry_id}-{device.index}"
         self._attr_device_info = coordinator.platform_device_info(device, device.type)
+        self._update_attributes()
 
-    @callback
-    def _handle_coordinator_update(self) -> None:
-        """Handle updated data from the coordinator."""
+    def _update_attributes(self) -> None:
+        """Update class attributes."""
```

Comment:
Why are we doing this change? Should this go in a separate PR since it switches logic?

Reply:
https://github.com/home-assistant/core/pull/140611